### DISCUSSION TITLE
스티커 기능

### DISCUSSION THREAD

**Discussion by IronKoh** [metamong-friends/metamong-friends.github.io#40] [제안하기]
다같이 회의를 할 때
보드에 포스트잇을 붙이면서 회의를 할 수 있었으면 좋겠습니다.
ProjectView팀에서는 디자인리뷰, IPM, 회고 시 마림바에서 해당 기능을 자주 사용했었습니다.

**Comment by ACT-TJ** [comment]
안녕하세요!!

저희도 향후 들어갈 Feature 로 많은 구성원들이 희망하고 있는데,
아시겠지만, 직접 개발을 해서 붙이는건 공수가 꽤 많이 드는 일입니다.
게더에서도 많은 써드파티 들을 연결, 활용하고 있는데 장,단점 들이 있을 것 같네요.
Miro 나 마림바(...있다면) 등을 연결하는 시나리오는 지속적으로 의견을 좁혀가고 있습니다.
Feature 개발 의견으로 관리하면서 저희도 플래닝 때 검토해 볼 수 있도록 하겠습니다.

이런게 있으면 좋겠다 하는 건 주저없이 의견 주시면 저희 올려주신 의견은 모두 취합하고 있으니
당장 기능에 반영하지 못하는 부분도 지속적으로 관리하도록 하겠습니다.

감사합니다!
